### PR TITLE
Branch protection

### LINKED ISSUE #10: Branch Protection
Would be good to start each site with branch protection rules.

Can this be set when the repo is initially created?

### PR DESCRIPTION
**Pull request by dkonieczek**
## Changes
- add branch protection for production branch upon creating new site via init
- Requires 'Snap Action' status check to succeed
- disables required reviewers

### CODE REVIEW

**Review by korgon — CHANGES_REQUESTED** [review]
Apparently I forgot this option - could you please add this into the PR? Thanks!
image

File: src/init.js [hunk 1]
```diff
@@ -178,7 +178,7 @@ export const init = async (options) => {
 		const { siteId, secretKey } = await auth.saveSecretKey(answers.secretKey, answers.siteId);
 
 		await setRepoSecret(options, { siteId: answers.siteId, secretKey: answers.secretKey, organization: answers.organization, name: answers.name });
-		console.log();
```
Comment (korgon) [inline src/init.js:181]: This was here for spacing - maybe return it or add some spacing another way?

See spacing missing now before "The testing-branch-protection directory...":
image

**Review by korgon — CHANGES_REQUESTED** [review]

**Comment by korgon** [comment]
Should we mention anything about checking out a new branch immediately since direct pushing to `production` is no longer accepted?

**Comment by dkonieczek** [comment]
- Added admins to be enforced by branch protection rule. 
- Improved the logging and it's spacing. Also mentioned to create a new branch.

**Review by korgon — CHANGES_REQUESTED** [review]
Looks good! One last thing...

```
&& git checkout -b branch
```

Since people 'may' copy and paste this - I think we should use something like `mockup` or `development` or something similar instead of `branch`.

**Review by korgon — APPROVED** [review]

**Pull request merged**